### PR TITLE
Update audio_transcription.py

### PR DESCRIPTION
**Pull request by RafalKaufman**
Modified the variable and method names in order to keep them 'self explanatory'.

### CODE REVIEW

**Comment by MichalKapala** [comment]
@RafalKaufman 
image

**Comment by RafalKaufman** [comment]
approve needed my lord

File: audio_transcription.py [hunk 1]
```diff
@@ -5,46 +5,47 @@
...
+        self.audio_path = audio_path
+        self.text_recognizer = sr.Recognizer()
+        self.dialogue_transcription = ""
 
-    def transcribe(self,filename):
-        with sr.AudioFile(filename) as source:
-            audio_listened = self.recognizer.record(source)
-            text = self.recognizer.recognize_google(audio_listened)
-        return text
+    def transcribe(self, audio_file_name):
```
Comment (MichalKapala) [inline audio_transcription.py]: As we are improving these code, I got one more improvement suggestion 

```suggestion
    def transcribe(self, audio_file_name : str) -> str:
```

As python language does not support strong typing, it's good to make such suggestion.
PS: Applicable also for other methods

File: audio_transcription.py [hunk 2]
```diff
@@ -5,46 +5,47 @@
...
             except sr.UnknownValueError as e:
                 print("Error:", str(e))
             else:
-                text = f"{text.capitalize()}. "
-                whole_text += text
-        return whole_text
+                chunk_transcription = f"{chunk_transcription.capitalize()}. "
+                whole_transcription += chunk_transcription
+        return whole_transcription
     def get_dialogue_transcription(self):
```
Comment (MichalKapala) [inline audio_transcription.py]: In general the `get` method  tells user that he should receive something from that method. I suggest to return self.transcription in this method and don't relay only on magic __str__ method

**Comment by MichalKapala** [comment]
@RafalKaufman Good improvements :+1:  Please be aware that my comment's are only suggestion and if you have some doubts feel free to start discussion ;)

**Comment by RafalKaufman** [comment]
@MichalKapala appreciate your comments. I've updated the file again. Could you spare a moment to give a **final review**?

**Comment by MichalKapala** [comment]
> @MichalKapala appreciate your comments. I've updated the file again. Could you spare a moment to give a **final review**?

I'd love to, but first check the code cause i see there is some syntax problem ;)

**Review by MichalKapala — APPROVED** [review]

**Pull request merged**